ukvm/ukvm_hv_kvm.h

```diff
@@ -30,6 +30,8 @@ struct ukvm_hvb {
     int vmfd;
     int vcpufd;
     struct kvm_run *vcpurun;
+    struct kvm_regs kregs;
```

Comment:
Would prefer not to add these two structures. These are just caches of the register state. It's much clearer to just get the register values directly from KVM using ioctls, and not stash/cache anything in `hv`. Otherwise we could get confused and just read or write to this `kregs` assuming that the values are up to date, or that writing to them will somehow make it to the actual VM registers.